src/app/modules/tabs/tabs.component.ts

```diff
@@ -67,6 +73,7 @@ export class AppTabsComponent implements OnInit {
 
     }
 
+    // Deprecated?: This may be redundant and/or unused. Seems like most of the handling uses id via (click) and passAction.
```

Comment:
can we remove this comment if we aren't sure if it's deprecated or not? Or if it is deprecated, can we remove the function 🧐

Reply:
removed